packages/forms-web-app/__tests__/unit/controllers/examination/enter-comment.test.js

```diff
@@ -2,43 +2,179 @@ const {
 	getEnterComment,
 	postEnterComment
 } = require('../../../../src/controllers/examination/enter-comment');
-const { mockReq, mockRes } = require('../../mocks');
+const { mockReq, mockRes, mockResponse } = require('../../mocks');
+
+const pathShortner = (obj, path) => path.split('.').reduce((value, el) => value && value[el], obj);
+const itemsPath = 'session.examination.selectedDeadlineItems.items';
 
 const pageData = {
 	backLinkUrl: '/examination/select-upload-evidence-or-comment',
-	id: 'examination-enter-comment',
-	pageTitle: 'Enter comment',
-	title: 'Enter comment'
+	id: 'examination-enter-a-comment',
+	pageTitle: 'Make a comment',
+	title: 'Make a comment',
+	hint: 'Any further information requested by',
+	captionTitle: 'Deadline item:',
+	comment: ''
 };
 
-describe('controllers/examination/submitting-for', () => {
+const hintValue = '`${pageData.hint} ${pathShortner(mockRequest, itemsPath)[0].submissionItem}`';
```

Comment:
this the same after eval for each hint. 

Can just be set to: "Any further information requested by Statement of Commonality of SoCG "

and remove this complexity

Reply:
done